pages/common/latexmk.md

```diff
@@ -0,0 +1,24 @@
+# latexmk
+
+> Compile LaTeX source files into finished documents. Does multiple runs when
+> the output will change.
```

Comment:
1) Put each sentence into its own line
2) The second sentence isn't clear. Can you rephrase it?